pkg/status/combinedstatus-resolution.go

```diff
@@ -264,6 +251,10 @@ func (c *combinedStatusResolution) generateCombinedStatus(bindingName string,
 		Results: make([]v1alpha1.NamedStatusCombination, 0, len(c.statusCollectorNameToData)),
 	}
 
+	if combinedStatus.Namespace == v1.NamespaceNone {
```

Comment:
I see that this PR did not introduce this choice, but: there are a lot of package paths that end with "v1". Using that alone as a name is very ambiguous. There is a very dominant tradition of using "metav1" as the name of this particular package. Not a blocker.